docs/dataprodukter/dele/dataprodukt.md

```diff
@@ -12,4 +12,19 @@ Der oppgis navn, beskrivelse, team i Teamkatalogen samt kontaktpunkt til teamet.
 Datasett legges til ved at man klikker seg inn på et `dataprodukt` på [Markedsplassen](https://data.intern.nav.no).
 Under `Legg til datasett` kan man registrere den aktuelle tabellen fra BigQuery sammen med metadata som gjør det enklere for teamet å beholde oversikten.
 Metadata gjør det også enklere for folk utenfor teamet å finne dataene.
-Beskrivelser av kolonnene i tabellen kan legges inn i [BigQuery-konsollet](https://console.cloud.google.com/bigquery).
\ No newline at end of file
+Beskrivelser av kolonnene i tabellen kan legges inn i [BigQuery-konsollet](https://console.cloud.google.com/bigquery).
+
+
+### Pseudonymisering av tabeller
```

Comment:
Burde vi kalle det dataset i stedet for tabeller? Syns teksten under er litt forvirrende fordi vi blander tabeller og BigQuery tabeller, som jo begge er det samme, men vi bruker forskjellige benevnelser.
Jeg tror det kan være fornuftig å skille på at vi lager pesudonymiserte datasett i et dataprodukt, som er representert i en BigQuery view. Dette vil også gjøre det enklere hvis vi ender opp med mer enn bare BigQuery.

Reply:
Dataset er samlingen tabeller/views i BQ, datasett på Markedsplassen blir heller ikke riktig fordi datasettet er den ferdig pseudonymiserte tabellen. For meg gir teksten mening.